lib/validate.js

```diff
@@ -37,7 +36,7 @@ module.exports = function makeValidatorMiddleware (middleware, schema, opts) {
   let validate
 
   function makeValidator () {
-    const reqSchema = merge({}, BASE_REQ_SCHEMA)
+    const reqSchema = structuredClone(BASE_REQ_SCHEMA)
```

Comment:
I guess I just wonder if a more simple approach to this would be enough? I am not opposed to this, but since we are not doing this for the specific reason `structuredClone` is meant to serve I wonder if there are any unexpected consequences of this approach.